cmd/kubeadm/app/cmd/options/generic.go

```diff
@@ -58,3 +61,14 @@ func AddControlPlanExtraArgsFlags(fs *pflag.FlagSet, apiServerExtraArgs, control
 func AddImageMetaFlags(fs *pflag.FlagSet, imageRepository *string) {
 	fs.StringVar(imageRepository, ImageRepository, *imageRepository, "Choose a container registry to pull control plane images from")
 }
+
+// AddFeatureGatesStringFlag adds the --feature-gates flag to the given flagset
+func AddFeatureGatesStringFlag(fs *pflag.FlagSet, featureGatesString *string) {
+	if knownFeatures := features.KnownFeatures(&features.InitFeatureGates); len(knownFeatures) > 0 {
+		fs.StringVar(featureGatesString, FeatureGatesString, *featureGatesString, "A set of key=value pairs that describe feature gates for various features. "+
+			"Options are:\n"+strings.Join(knownFeatures, "\n"))
+	} else {
+		fs.StringVar(featureGatesString, FeatureGatesString, *featureGatesString, "A set of key=value pairs that describe feature gates for various features. "+
+			"No gates are available in this release.")
```

Comment:
one final nit: please change to `No feature gates are available in this release.`